types/scene/wlr_scene.c

```diff
@@ -276,6 +276,12 @@ static void scene_node_opaque_region(struct wlr_scene_node *node, int x, int y,
 		if (scene_rect->color[3] != 1) {
 			return;
 		}
+		if (!wlr_box_empty(&scene_rect->clipped_region.area)) {
+			struct wlr_box *clipped = &scene_rect->clipped_region.area;
+			pixman_region32_intersect_rect(opaque, opaque, clipped->x, clipped->y,
+					clipped->width, clipped->height);
```

Comment:
wouldn't we want to do the opposite? The clipped region is not opaque

Reply:
I tested with the opposite and the rendering still seemed to work fine, I'll wait for you to respond though in case I'm missing something!

Reply:
Good catch!